src/CLI.hs

```diff
@@ -223,114 +218,32 @@ hevm actspec cid sol' solver' timeout _ = do
   let act = validation (\_ -> error "Too bad") id (enrich <$> compile specContents)
   bytecode <- fmap fromJust $ solcRuntime cid solContents
   let actbehvs = translateAct act
-  sequence_ $ flip fmap actbehvs $ \(behvs,calldata) ->
+  sequence_ $ flip fmap actbehvs $ \(name,behvs,calldata) ->
     Solvers.withSolvers solver' 1 (naturalFromInteger <$> timeout) $ \solvers -> do
-    
       solbehvs <- removeFails <$> getBranches solvers bytecode calldata
+      putStrLn $ "\x1b[1mChecking behavior \x1b[4m" <> name <> "\x1b[m of Act\x1b[m"
       -- equivalence check
-      putStrLn "Checking if behaviours are equivalent"
-      checkResult =<< equivalenceCheck' solvers solbehvs behvs debugVeriOpts
-      -- exhaustiveness sheck
-      putStrLn "Checking if the input space is the same"
+      putStrLn "\x1b[1mChecking if behaviour is matched by EVM\x1b[m"
+      checkResult =<< checkEquiv solvers debugVeriOpts solbehvs behvs
```

Comment:
```suggestion
      checkResult =<< checkEquiv solvers defaultVeriOpts solbehvs behvs
```

Reply:
It's fixed in #156! I will not change it here to avoid merge conflicts.